man/systemd.scope.xml

```diff
@@ -46,14 +46,14 @@
     Control Group Interfaces</ulink> for an introduction on how to make
     use of scope units from programs.</para>
 
-    <para>Note that unlike service units scope units have no "main" process, all processes in the scope are
-    equivalent. The lifecycle of the scope unit is thus not bound to the lifetime of one specific process but
-    to the existance of any processes in the scope. This also means that the exit status of these processes
-    do not cause the scope unit to enter a failure state. Scope units may still enter a failure state, for
-    example due to resource exhaustion or stop timeouts being reached, but not due to programs inside of them
-    terminating uncleanly. Since processes managed as scope units generally remain children of the original
-    process that forked them off it's also the job of that process to collect their exit statuses and act on
-    them as needed.</para>
+    <para>Note that unlike service units, scope units have no "main" process, i.e. all processes in the scope
```

Comment:
I think `Note that, unlike service units, scope units have no "main" process: all processes in the scope are equivalent.` reads better.